service/src/main/java/org/folio/rest/workflow/kafka/EventProducer.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.rest.workflow.kafka;
+
+import org.folio.spring.messaging.model.Event;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EventProducer {
+
+  private static final Logger logger = LoggerFactory.getLogger(EventProducer.class);
+
+  @Value("${application.kafka.producer.events.topic}")
+  private String topic;
+
+  private KafkaTemplate<String, Event> eventTemplate;
+
+  @Autowired
+  public EventProducer(KafkaTemplate<String, Event> eventTemplate) {
+    this.eventTemplate = eventTemplate;
+  }
+
+  public void send(Event event) {
+    logger.info("Send []: {}, {}", event.getMethod(), event.getPath());
```

Comment:
```suggestion
    logger.debug("Send []: {}, {}", event.getMethod(), event.getPath());
```
I don't know but we might want this set to `debug`.
It seems like the logging here might be excessive.

Reply:
Done.